src/main/java/org/jitsi/videobridge/rest/JSONSerializer.java

```diff
@@ -425,17 +424,15 @@ public static JSONObject serializeChannelCommon(
                         ColibriConferenceIQ.Channel.ID_ATTR_NAME,
                         id);
             }
-            // channelBundleId
-            if (channelBundleId != null)
-            {
-                jsonObject.put(
-                        ColibriConferenceIQ.ChannelCommon
-                                .CHANNEL_BUNDLE_ID_ATTR_NAME,
-                        channelBundleId);
-            }
             // endpoint
             if (endpoint != null)
             {
+                // channelBundleId
+                jsonObject.put(
+                    ColibriConferenceIQ.ChannelCommon
+                        .CHANNEL_BUNDLE_ID_ATTR_NAME,
+                    endpoint);
+
```

Comment:
Why do we need this last one? I think the JSON-XML translation should be kept as simple as possible. Any backward-compat stuff should be handled in another layer.